okapi-core/src/main/java/org/folio/okapi/managers/ProxyService.java

```diff
@@ -954,19 +959,38 @@ private void proxyInternal(Iterator<ModuleInstance> it,
     );
   }
 
+  static Buffer compressInternalResponse(RoutingContext ctx, Buffer respBuf) {
+    String acceptEncoding = ctx.request().getHeader("Accept-Encoding");
+    int sz = respBuf.length();
+    if (acceptEncoding != null && acceptEncoding.contains("gzip")
+        && sz > CONTENT_COMPRESS_MIN_SIZE) {
+      try {
+        ByteArrayOutputStream dst = new ByteArrayOutputStream();
+        GZIPOutputStream src = new GZIPOutputStream(dst);
```

Comment:
Sorry I'm on a roll now... :) Maybe `src` should be something like `gzipedOutputStream` or `compressedOutput` or similar.

Reply:
ok. I've renamed them now